src/commands/fun/xkcd.js

```diff
@@ -0,0 +1,29 @@
+const SA = require('superagent')
+const Command = require('../../classes/Command')
+
+module.exports = new Command(async (msg, suffix) => {
+  const res = await SA.get('http://xkcd.com/info.0.json')
+  if(res.error) {
+    msg.channel.createMessage(`<@${msg.author.id}>, Please try again later.`)
```

Comment:
Be more descriptive, "try again later" on its own is way too ambiguous to end users